src/Dynamic/tasks.pug

```diff
@@ -0,0 +1,11 @@
+extends layout
+block content
+    section.section
+        .container
+            .hero
+                .hero-body.hero-slim
+                    .has-text-centered
+                        h1.title.is-size-4 #{__('site_name')} Tasks
+            include Includes/tasks
```

Comment:
I don't understand why this is an include? It only gets used here and having it as an include increases the render time?

Reply:
No, I tried to keep the structure used by lists. It should not affect the render time.

Reply:
Huh, hadn't even realised lists was structured like that. Project consistency aside, which makes more sense to you?

Reply:
It was misinterpreted by me, lists was only there because it was used by multiple routes.